pages/admin/admin-management.tsx

```diff
@@ -262,8 +268,43 @@ export default function AdminManagement() {
       );
       setRecordsCount(data.employees.totalCount);
     },
+    onError: error => {
+      toast({
+        status: 'error',
+        description: error.message,
+      });
+    },
   });
 
+  // Delete Employee Hook
+  const [deleteEmployee] = useMutation<DeleteEmployeeResponse, DeleteEmployeeRequest>(
+    DELETE_EMPLOYEE_MUTATION,
+    {
+      onCompleted: data => {
+        if (data.deleteEmployee.ok) {
+          toast({
+            status: 'success',
+            description: `${userToDelete?.name} has been deleted`,
```

Comment:
No longer use `userToDelete`. Use the `employee` from the `deleteEmployee` mutation result